src/BasePlatform.js

```diff
@@ -79,10 +79,12 @@ export default class BasePlatform {
      * with getUserMedia, return a string explaining why not.
      * Otherwise, return null.
      */
-    screenCaptureErrorString() {
+    screenCaptureErrorString(): string {
         return "Not implemented";
     }
 
+    isElectron(): boolean { return false; }
```

Comment:
or since now they have getHumanReadableName could do it based off of that but that feels dirty